.env.testnet.qa

```diff
@@ -17,9 +17,6 @@ NEXT_PUBLIC_GENERAL_BUCKET_ADDRESS=0x72Ed7d7b7835Ad62B1f9b6280bAd62618aA71461
 NEXT_PUBLIC_CHAIN_ID=31
 
 # CR-related env variables
-# TODO: To be removed 
-NEXT_PUBLIC_SIMPLIFIED_REWARD_DISTRIBUTOR_ADDRESS=0xc469Cc2579De5C16210e9063B4E628bF8C46bA02
```

Comment:
I'd put everything that's not pertinent to this PR in another one as it's already quite large